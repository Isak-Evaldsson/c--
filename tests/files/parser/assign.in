int test() {
    test_var = 10;
}