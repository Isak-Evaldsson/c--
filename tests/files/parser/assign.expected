Prototype: test
    Assignment: test_var
        LiteralExpr: 10
